js/two-factor.js

```diff
@@ -0,0 +1,35 @@
+/* global ajaxurl */
```

Comment:
this is another of those trying to switch between two coding standards on the fly.